docs/guides/release-notes.md

```diff
@@ -12,11 +12,37 @@ Track all Aviatrix Terraform provider's releases. New resources, features, and b
 
 ---
 
-``Last updated: R3.1.6 (UserConnect-7.1.4183)``
+``Last updated: R3.2.0 (UserConnect-7.2.4820)``
 
 ---
 
-## 3.1.6 (Unreleased)
+## 3.2.0
+### Notes:
+- Supported Controller version: **UserConnect-7.2.4820**
```

Comment:
Please add the release date as well for both 3.2.0 and 3.1.6 sections under the ### Notes:

You can check the older releases for reference format